### PR TITLE
Add tensorflow_transform.__version__ (fixes #94)

### LINKED ISSUE #94: Add tensorflow_transform.__version__
Hi! For running Dataflow jobs it is sometimes required for us to ensure that Dataflow workers have certain required packages installed. For this reason, it would be useful to be able to determine the `tensorflow_transform` version at runtime.

Ideally, this would follow the common python convention of a `__version__` in the top-level module. For example, how `tensorflow-data-validation` does:

```
In [1]: import tensorflow_data_validation
In [2]: tensorflow_data_validation.__version__
Out[2]: '0.11.0'
```

### PR DESCRIPTION
**Pull request by brianmartin**
See #94 for detail.

### CODE REVIEW

File: tensorflow_transform/version.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Contains the version string of TF.Transform."""
+
+# Note that setup.py uses this version.
+__version__ = '0.12.0dev'
```
Comment (zoyahav) [inline tensorflow_transform/version.py]: Please update to 0.13.0dev

Comment (brianmartin) [reply tensorflow_transform/version.py]: 👍

**Comment by brianmartin** [comment]
Sure thing. Just rebased!

**Comment by brianmartin** [comment]
Ah, yes. Updated `RELEASE.md` as well.

**Pull request merged**